### PR TITLE
[View api] Setting up tests in GitHub actions CI/CD

### PR DESCRIPTION
**Pull request by NoahNsimbe**
**_WHAT DOES THIS PR DO?_**

- [x] Setup for view API tests
- [x] Adds more tests to the view API

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [MOB-85](https://airqoteam.atlassian.net/browse/MOB-85)
    - [MOB-9](https://airqoteam.atlassian.net/browse/MOB-9)

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

File: .github/workflows/run-tests-production.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+name: run-tests
+
+on:
+  pull_request:
+    branches:
+      - master
+
```
Comment (NoahNsimbe) [inline .github/workflows/run-tests-production.yml:7]: production test file

File: .github/workflows/run-tests-staging.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
+name: run-tests
+
+on:
+  pull_request:
+    branches:
+      - staging
```
Comment (NoahNsimbe) [inline .github/workflows/run-tests-staging.yml:6]: staging test file

**Review by 123MwanjeMike — APPROVED** [review]
LGTM

**Review by Baalmart — APPROVED** [review]
LGTM, thanks @NoahNsimbe

**Pull request merged**